code/game/objects/items/weapons/extinguisher.dm

```diff
@@ -160,5 +167,11 @@
 /obj/item/weapon/extinguisher/proc/EmptyExtinguisher(var/mob/user)
 	if(loc == user && reagents.total_volume)
 		reagents.clear_reagents()
+
+		var/turf/T = get_turf(loc)
+		if(istype(T, /turf/open))
+			var/turf/open/theturf = T
+			theturf.MakeSlippery(TURF_WET_WATER)

```

Comment:
Noted. Tbh though, I didn't even know you could alt click extinguishers to empty them.